### ISSUE TITLE
Luxmed API exception: Session expired

### ISSUE THREAD

**Issue by marska** [dyrkin/luxmed-bot#70]
When run "Find terms" I get error "Luxmed API exception: Session expired".

**Comment by bartlomiejborzucki** [comment]
I have the same ;/

**Comment by dyrkin** [comment]
Most likely it is yet another API change. Don't have time to investigate the issue now :( It'd be appreciated if someone fixes this

**Comment by msobecki** [comment]
I have made some changes on https://github.com/msobecki/luxmed-bot/tree/jwt-token-support to fix this issue. It seems that some of the endpoints require JWT token to authorize requests. Currently I'm searching for it in JS code of reservation page as a ugly hack, because I don't know if it is possible to get it directly from API in JSON format. I have also updated some of headers sent to LuxMed API by bot based on traffic send by official Android app. 

I'm not very familiar with technical details of communication between bot and LuxMed API and I'm not really into Scala, just made this quick fix for my own needs. Feel free to make pull request if my changes looks good for you.
